extensions/extensions-docker/aissemble-airflow/src/main/resources/docker/Dockerfile

```diff
@@ -5,9 +5,7 @@ LABEL org.opencontainers.image.source = "https://github.com/boozallen/aissemble"
 
 USER root
 
-COPY ./target/cacerts/* /usr/local/share/ca-certificates/
-
-RUN update-ca-certificates && apt-get update && apt-get install -y --no-install-recommends apt-utils
```

Comment:
Isn't this gonna make the images unbuildable from a BAH device since they needs the certs?

Reply:
If you log out of your VPN to do the build you should be able to build just fine.  But we can't really install certs specific to one organization. I think long-term it'd be nice to find some sort of just-in-time solution that allows VPN CA certs to be made available without actually modifying the final image.